tests/testthat/test-data_type.R

```diff
@@ -15,7 +15,8 @@ test_that("identify_data_type and split_raw_data works as expected", {
   colnames(df_2)[colnames(df_2) %in% c(colnames(drugs), "Concentration")] <- 
     paste0(colnames(df_2)[colnames(df_2) %in% c(colnames(drugs), "Concentration")], "_2")
   
-  df_layout_2 <- merge(df_layout, df_2, allow.cartesian = TRUE)
+  df_layout_2 <- df_layout[df_2, on = intersect(names(df_layout), names(df_2)),
+                           allow.cartesian = TRUE]
```

Comment:
Just a thought - maybe a better solution would be to use `colnames` instead of `names`, even if it works (just to be _explicit_ How you think?
e.g. `intersect(colnames(df_layout), colnames(df_2))` inseatd of `intersect(names(df_layout), names(df_2))`

Reply:
🤔